src/ripple/app/misc/impl/AmendmentTable.cpp

```diff
@@ -326,31 +362,87 @@ AmendmentTableImpl::AmendmentTableImpl(
         }
     }
 
+    hash_set<uint256> detect_conflict;
+    // Parse enabled amendments from config
     for (auto const& a : parseSection(enabled))
     {
-        if (auto s = add(a.first, sl))
-        {
-            JLOG(j_.debug()) << "Amendment " << a.first << " is enabled.";
-
-            if (!a.second.empty())
-                s->name = a.second;
-
-            s->supported = true;
-            s->enabled = true;
+        if (FeatureVotesExist)
+        {  // If the table existed, warn about duplicate config info
+            JLOG(j_.warn()) << "[amendments] section in config file ignored"
+                               " in favor of data in db/wallet.db.";
+            break;
+        }
+        else
+        {  // Otherwise transfer config data into the table
+            detect_conflict.insert(a.first);
+            persistVote(a.first, a.second, false);  // un-veto
         }
     }
 
+    // Parse enabled amendments from config
     for (auto const& a : parseSection(vetoed))
```

Comment:
Comment is wrong. Should say "Parse vetoed..."

Reply:
Done.